homeassistant/helpers/entity_registry.py

```diff
@@ -612,6 +620,16 @@ def async_get_entity_id(
         """Check if an entity_id is currently registered."""
         return self.entities.get_entity_id((domain, platform, unique_id))
 
+    @callback
+    def async_entity_ids(self) -> list[str]:
```

Comment:
Same for these.

Reply:
Removed in https://github.com/home-assistant/core/pull/112311